src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -3470,6 +3470,29 @@ public void CS0246ERR_SingleTypeNameNotFound06()
                 );
         }
 
+        [Fact, WorkItem(69700, "https://github.com/dotnet/roslyn/issues/69700")]
+        public void CS0246ERR_SingleTypeNameNotFound07()
+        {
+            var text =
+@"class C
+{
+    [SomeAttribute<int>]
+    static void Foo()
```

Comment:
>Foo [](http://example.com/codeflow?start=16&length=3)

Sorry, I missed that in first go. "Foo" is disallowed in the codebase because it triggers some internal word check rules.